lam4doc/entrypoints/ui/api_wrapper.py

```diff
@@ -13,21 +13,24 @@
 import requests
 from requests import Timeout
 
-from lam4doc import config
-from lam4doc.config import LAM_LOGGER
+from lam4doc.config import LAMConfig
 
-logger = logging.getLogger(LAM_LOGGER)
+config = LAMConfig()
+logger = logging.getLogger(config.LAM_LOGGER)
 
 
-def get_lam_report() -> tuple:
+def get_lam_report(report_extension: str) -> tuple:
     """
     Method to connect to the lam api to get the report
+    :type report_extension: report extension type
     :return: html report
     :rtype: file, int
     """
-    logger.debug('start get report api call')
+    logger.debug(f'start get report api call with {report_extension} format')
     try:
-        response = requests.get(url=config.LAM_API_SERVICE + '/generate-report', timeout=config.LAM_DEFAULT_TIMEOUT)
+        response = requests.get(url=config.LAM_API_SERVICE + '/generate-report',
```

Comment:
is there a safer way of handling URL concatenation? Perhaps `from urllib.parse import urljoin` is more reliable. 
(This applies to all URI handling; as this is the counterpart to the Path handling which we consistently do with pathlib)